src/bindgen/config.rs

```diff
@@ -765,6 +765,8 @@ pub struct Config {
     pub documentation: bool,
     /// How documentation comments should be styled.
     pub documentation_style: DocumentationStyle,
+    /// Optional attribute to apply to pointers that are required to not be null
+    pub non_null_attribute: Option<String>,
```

Comment:
I think rather than it being in the top level section it may be worth introducing a `[ptr]` section of sorts like the `ConstantConfig` and such. It just feels a little out of place... wdyt?

Reply:
done